src/gluonts/dataset/common.py

```diff
@@ -158,18 +164,18 @@ class FileDataset(Dataset):
 
     def __init__(
         self,
-        path: Path,
+        path: Union[str, Path, List[Path], List[str]],
```

Comment:
What's the use case for supporting multiple paths?

Reply:
> These are some great numbers! Have you tried caching the dataset as an alternative benchmark?

Yes, if data is in memory samples/s is much higher (around 1.0E7). The main purpose of the ArrowDataset is for large datasets that you cannot cache in memory.

Reply:
@vafl Multiple files are already supported. No need to supply multiple paths?